resources/process_workflow.md

```diff
@@ -35,6 +35,28 @@ Be sure not to create a task such as "create a input field", or "create an API",
 
 All SMART tasks should be linked to a User Story, which should be properly detailed with its goal (it can be written using the User Story pattern), Acceptance criteria, and BDD if necessary.
 
+We are following Gherkin syntax to describe our User Stories:
+
+### Title:
+```
```

Comment:
**suggestion:** you should add a linebreak after a Markdown title

Reply:
Done in Done in ba05e0e